pkg/k8s/wait.go

```diff
@@ -169,9 +169,9 @@ func isReady(name string) ConditionFunc {
 		allReady := true
 		for _, c := range obj.Status.Conditions {
 			if !c.IsTrue() {
-				msg := fmt.Sprintf("%s is not %s, %s: %s", name, c.Type, c.Reason, c.Message)
+				msg := fmt.Sprintf("%s is not %s\n\nResource: %s\n", name, ready.Type, resource(obj))
```

Comment:
Is there a reason you're no longer logging the individual condition that's not true? you changed `c.Type` to `ready.Type` maybe that's just cope pasted from the previous logging?

Reply:
Yeah, no good reason just copy-paste.

```suggestion
				msg := fmt.Sprintf("%s is not %s, %s: %s\n\nResource: %s\n", name, c.Type, c.Reason, c.Message, resource(obj))
```

Reply:
Done